cmd/config-versions.go

```diff
@@ -936,3 +936,47 @@ type serverConfigV33 struct {
 		// Add new external policy enforcements here.
 	} `json:"policy"`
 }
+
+// serverConfigV34 is just like version '32', removes clientID from MQTT.
```

Comment:
Shouldn't the comment describe how V34 differs from V33?

Reply:
changed it